### PR TITLE
References for UVL

### PR DESCRIPTION
**Pull request by SundermannC**
This pull request contains the added references for the camera ready version of both UVL papers.

### CODE REVIEW

File: literature-cleaned.bib [hunk 1]
```diff
@@ -1,3 +1,13 @@
+@inproceedings{SHE+:MODEVAR21,
+  author = {Sundermann, Chico and Heß, Tobias and Engelhardt, Dominik and Arens, Rahel and Herschel, Johannes and Jedelhauser, Kevin and Jutz, Benedikt and Krieter, Sebastian and Schaefer, Ina},
+  title = {{Integration of UVL in FeatureIDE}},
+  year = 2021,
+  publisher = ACM,
+  booktitle = MODEVAR,
+  series = {SPLC '21},
```
Comment (pmbittner) [inline literature-cleaned.bib:7]: We usually drop series.

File: literature-cleaned.bib [hunk 2]
```diff
@@ -55,6 +65,14 @@ @inproceedings{KTS:FormaliSE21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@thesis{A21,
```
Comment (pmbittner) [inline literature-cleaned.bib:68]: We use `@mastersthesis` here. Is `@thesis` a valid tag? If so, do we want to change `@mastersthesis` for all bachelor theses to `@thesis`?

File: literature-cleaned.bib [hunk 3]
```diff
@@ -55,6 +65,14 @@ @inproceedings{KTS:FormaliSE21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@thesis{A21,
+ author = {Arens, Rahel},
+  title = {Incremental Construction of Modal Implication Graphs for Feature-Model Evolution},
+  year = 	2021,
+  school = {TU Braunschweig},
+	type = {Bachelor's Thesis},
```
Comment (pmbittner) [inline literature-cleaned.bib:73]: Weird indents.

File: literature-cleaned.bib [hunk 4]
```diff
@@ -1884,6 +1912,15 @@ @article{DBW:SCP18
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2017 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+
+@techreport{BFT17,
+  author = {Clark Barrett and Pascal Fontaine and Cesare Tinelli},
+  title =	{{The SMT-LIB Standard: Version 2.6}},
+  institution =	{Department of Computer Science, The University of Iowa},
+  year = 2017
+}
```
Comment (pmbittner) [inline literature-cleaned.bib:1921]: weird indents

File: literature-cleaned.bib [hunk 5]
```diff
@@ -2526,6 +2563,19 @@ @inproceedings{FMS+:SANER17
...
+@inproceedings{PRS+:WWW16,
+  author = {Pezoa, Felipe and Reutter, Juan L. and Suarez, Fernando and Ugarte, Mart\'{\i}n and Vrgo\v{c}, Domagoj},
+  title = {{Foundations of JSON Schema}},
+  year = 2016,
+  publisher = {International World Wide Web Conferences Steering Committee},
+  booktitle = {Proceedings of the 25th International Conference on World Wide Web},
+  pages = {263–273},
+  numpages = {11},
+  keywords = {expressiveness of schema languages, JSON, JSON validation, JSON schema},
+  series = {WWW '16}
```
Comment (pmbittner) [inline literature-cleaned.bib:2575]: `- series`

File: literature-cleaned.bib [hunk 6]
```diff
@@ -2526,6 +2563,19 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PRS+:WWW16,
+  author = {Pezoa, Felipe and Reutter, Juan L. and Suarez, Fernando and Ugarte, Mart\'{\i}n and Vrgo\v{c}, Domagoj},
+  title = {{Foundations of JSON Schema}},
+  year = 2016,
+  publisher = {International World Wide Web Conferences Steering Committee},
+  booktitle = {Proceedings of the 25th International Conference on World Wide Web},
+  pages = {263–273},
```
Comment (pmbittner) [inline literature-cleaned.bib:2572]: --

File: literature-cleaned.bib [hunk 7]
```diff
@@ -10429,6 +10497,18 @@ @inproceedings{KTS+:ICSE09
...
 
+@inproceedings{H:DLS08,
+  author = {Hickey, Rich},
+  title = {{The Clojure Programming Language}},
+  year = 2008,
+  publisher = ACM,
+  booktitle = {Proceedings of the 2008 Symposium on Dynamic Languages},
+  articleno = {1},
+  numpages = {1},
+  series = {DLS '08}
```
Comment (pmbittner) [inline literature-cleaned.bib:10508]: `- series`

File: literature-cleaned.bib [hunk 8]
```diff
@@ -2526,6 +2563,19 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PRS+:WWW16,
+  author = {Pezoa, Felipe and Reutter, Juan L. and Suarez, Fernando and Ugarte, Mart\'{\i}n and Vrgo\v{c}, Domagoj},
+  title = {{Foundations of JSON Schema}},
+  year = 2016,
+  publisher = {International World Wide Web Conferences Steering Committee},
+  booktitle = {Proceedings of the 25th International Conference on World Wide Web},
```
Comment (pmbittner) [inline literature-cleaned.bib:2571]: As we usually reuse booktitle values with variables, they are not year-specific. So I would suggest to drop the `25th` for consistency, and optionally to extract into variable.

File: literature-cleaned.bib [hunk 9]
```diff
@@ -10429,6 +10497,18 @@ @inproceedings{KTS+:ICSE09
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2008 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{H:DLS08,
+  author = {Hickey, Rich},
+  title = {{The Clojure Programming Language}},
+  year = 2008,
+  publisher = ACM,
+  booktitle = {Proceedings of the 2008 Symposium on Dynamic Languages},
```
Comment (pmbittner) [inline literature-cleaned.bib:10505]: see comments on `PRS+:WWW16`

File: literature-cleaned.bib (hunk 8 above)
Comment (pmbittner) [reply literature-cleaned.bib:2571]: For further consistency, please abbreviate `Proceedings` etc as in the values in MYabrv:
`@String{ICSE = "Proc.\ Int'l Conf.\ on Software Engineering (ICSE)"}`

File: literature-cleaned.bib [hunk 10]
```diff
@@ -13234,6 +13314,15 @@ @article{BSR04
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2003 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 
+@book{F03,
+  title = {{UML Distilled: A Brief Guide to the Standard Object Modeling Language (3rd Edition)}},
+  author = {Fowler, Martin},
+  edition = 3,
+  publisher = {Addison-Wesley Professional},
```
Comment (pmbittner) [inline literature-cleaned.bib:13321]: Can we reuse `AddisonWesleyLongman` variable here or is it a bad match?

**Review by pmbittner — CHANGES_REQUESTED** [review]
Uuuughhh, I accidentally commented on `literature-cleaned.bib`. The comments should apply anyway though.

File: literature-cleaned.bib (hunk 2 above)
Comment (SundermannC) [reply literature-cleaned.bib:68]: I changed it to `@mastersthesis` for now.

File: literature-cleaned.bib (hunk 10 above)
Comment (SundermannC) [reply literature-cleaned.bib:13321]: I was not sure on that. After further research, I think it using the variable should be correct.

Comment (pmbittner) [reply literature-cleaned.bib]: Ok, proceed as you like. :)

**Pull request merged**